src/odemis/gui/cont/acquisition/overview_stream_acq.py

```diff
@@ -101,3 +101,65 @@ def open_acquisition_dialog(self):
             return acq_dialog.data
         else:
             return None
+
+
+class CorrelationDialogController(object):
+    """Controller to handle the multipoint correlation (3DCT) initialization"""
+
+    def __init__(self, tab_data, tab):
+        """
+        tab_data (MicroscopyGUIData): the representation of the microscope GUI
+        tab: (Tab): the tab which should show the data
+        """
+        self._tab_data_model = tab_data
+        self._main_data_model = tab_data.main
+        self._tab = tab
+
+    def open_correlation_dialog(self):
+        """
+        return None or a list of DataArrays: the acquired images. None if it was
+          cancelled.
+        """
+        # Indicate we are acquiring, especially important for the SEM which
+        # need to get the external signal to not scan (cf MicroscopeController)
+        self._main_data_model.is_acquiring.value = True
```

Comment:
Is it really important here? The dialog is modal, and all the streams are paused. Do you also need this?